lib/screens/v2/candidate_generator/pre_fare.ex

```diff
@@ -20,16 +20,27 @@ defmodule Screens.V2.CandidateGenerator.PreFare do
             ],
             screen_takeover_left: [
               :full_screen_left
-            ]
+            ],
+            body_takeover_left: [:full_body_left]
```

Comment:
This still needs the header, I think.  Could be consolidated with `screen_normal_left`, maybe?

Reply:
Good catch. Changed up the left template to allow for that.